src/httprpc.h

```diff
@@ -7,6 +7,7 @@
 
 #include <string>
 #include <map>
+#include <set>
```

Comment:
Why has that to be here?

Reply:
Wasn't sure, was just trying to follow local style. Not sure that string or map need to be there either?

Reply:
None needs to be there. If you want to remove, do it in a different commit.